doc/source/terrain.md

```diff
@@ -408,6 +411,32 @@ fractal_roughness = dem.fractal_roughness()
 fractal_roughness.plot(cmap="Reds", cbar_title="Fractal roughness (dimensions)")
 ```
 
+(texture-shading)=
+## Texture shading
+
+{func}`xdem.DEM.texture_shading`
+
+The texture shading technique produces an isotropic relief visualization that emphasizes the drainage network structure of terrain, including ridges and canyons. Unlike traditional hillshading, texture shading exhibits scale invariance and orientation independence, making it particularly effective for revealing the hierarchical structure of mountainous topography without directional bias.
+
+Texture shading is computed by applying a fractional Laplacian operator to the elevation data, which acts as a scale-independent high-pass filter. The technique produces relative elevation values where positive values correspond to ridges and peaks (higher than surrounding terrain) and negative values correspond to valleys and canyons (lower than surrounding terrain). The method includes a detail parameter α (typically 0.5-1.0) that controls the emphasis given to fine terrain features versus major landscape structure.
```

Comment:
This makes me think a bit about the TPI, also implemented in xDEM. I'm curious how both would compare on the same terrain, but TPI is definitely not scale-independent.